main.go

```diff
@@ -185,6 +189,8 @@ func main() {
 	}()
 
 	BoshMaintenanceLoop := time.NewTicker(1 * time.Hour)
+	redoLogin := time.NewTicker(23 * time.Hour)
```

Comment:
Is this a thing that needs to be configurable?

Reply:
it possible that someone that has UAA auth enabled on the director could set the relogin time to be less than 24 hrs.

Reply:
Can we go ahead and make that configurable in Blacksmith?

Reply:
sure.  do we want to just add another yaml line for this called timeout or something similar?

Reply:
Yes

Reply:
Yes

Reply:
Another user (@bruxelles86) ran across this same problem, and would also benefit from a configurable timeout.  Is that change in the works?

Reply:
i'll revisit this weekend.  i have some new thoughts on how to get around some of the issues i encountered when first doing this

Reply:
Did you ever get back to this, @anishp55 ?

Reply:
no, i'll try to get to it this week.